### PR TITLE
[MPQEditor] Implement `handleLoadVisqFile`

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit implements processing of 'loadVisqFile' message and 'showVisqCircleModelGraph' function needed for that.

Its correctness is tested in https://github.com/Samsung/ONE-vscode/pull/1543

Fresh draft: https://github.com/Samsung/ONE-vscode/pull/1543
Full draft: https://github.com/Samsung/ONE-vscode/pull/1505
Related: https://github.com/Samsung/ONE-vscode/issues/1491

ONE-vscode-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by jyoungyun — DISMISSED** [review]
LGTM

File: src/MPQEditor/MPQEditor.ts [hunk 1]
```diff
@@ -716,4 +740,32 @@ export class MPQEditorProvider
...
+   */
+  private handleLoadVisqFile(
+    document: vscode.TextDocument,
+    webviewPanel: vscode.WebviewPanel
+  ) {
+    const dialogOptions = {
+      canSelectMany: false,
+      canSelectFolders: false,
+      openLabel: "Open",
+      filters: { "target files": ["visq.json"], "all files": ["*"] },
```
Comment (dayo09) [inline src/MPQEditor/MPQEditor.ts:755]:
```suggestion
      filters: { "target files": ["visq.json"], "all files": ["*"]},
      title: "Open VISQ File" };
```

Comment (dayo09) [reply src/MPQEditor/MPQEditor.ts:755]: Without this, the system shows

> "" is ready.

In the dialog.

Comment (stamalakhov) [reply src/MPQEditor/MPQEditor.ts:755]: Ahhh. Sorry. Thank you very much!

**Review by dayo09 — APPROVED** [review]
LGTM Thanks!

**Review by jyoungyun — APPROVED** [review]
LGTM

**Comment by dayo09** [comment]
@Samsung/one-vscode_committers PTAL :-D

**Pull request merged**